docs/tutorial.rst

```diff
@@ -101,7 +101,7 @@ A test must also define a validation function which is decorated with the :func:
 This function will be used to validate the test's output after it is finished.
 ReFrame, by default, makes no assumption about whether a test is successful or not;
 it is the test's responsibility to define its validation.
-The framework provides a rich set of `utility functions <deferrable_functions_reference>`__ that help matching patterns and extract values from the test's output.
+The framework provides a rich set of :doc:`utility functions <deferrable_functions_reference>`__ that help matching patterns and extract values from the test's output.
```

Comment:
```suggestion
The framework provides a rich set of :doc:`utility functions <deferrable_functions_reference>` that help matching patterns and extract values from the test's output.
```

Reply:
@vkarak Yeah I can. Do you want me to see what other links are broken and fix them?

Reply:
That would be welcome.

Reply:
Okay I went ahead and did that.